src/commands/createContainerApp/createContainerApp.ts

```diff
@@ -32,10 +32,11 @@ export async function createContainerApp(context: IActionContext & Partial<ICrea
     };
 
     const title: string = localize('createContainerApp', 'Create Container App');
+    const useQuickStartImage: boolean = true;
 
     const promptSteps: AzureWizardPromptStep<IContainerAppWithActivityContext>[] = [
         new ContainerAppNameStep(),
-        new ImageSourceListStep(),
+        new ImageSourceListStep({ useQuickStartImage }),
```

Comment:
I think it would be nice to not have to pass the optional parameter when trying to obtain the default behavior.  If we change the options naming to something like `hideQuickStartImage` instead of `useQuickStartImage`, we are then able to show the option by default even when no options parameter is passed

Reply:
I actually prefer the opposite way-- we should have to opt-in to showing the quickstart image. Right now, we show/hide it 50% of the time in either direction, but I feel like there are likely more scenarios where we would hide the image rather than display it.

But I think either way, it doesn't really impact the code that much. It's a half-full half-empty scenario 🥤

Reply:
Agreed, I had actually already talked with Megan and we decided on doing what you just mentioned.  I originally had the file under the create folder, but we ended up moving it to the deploy folder anyway, so it makes more sense to show instead of hide 🙂